proto/storage/policy.proto

```diff
@@ -76,7 +76,7 @@ message Policy {
   // Read-only field. Indicates the policy is a default policy if true and a custom policy if false.
   bool is_default = 26; // @gotags: crYaml:"isDefault"
 
-  PolicySource source = 27; // @gotags: crYaml:"-"
+  PolicySource source = 27; // @gotags: search:"Policy Source,store" crYaml:"-"
```

Comment:
Why do we need a search tag?

Reply:
Now time for some bad news.  The column will not be populated.  To populate it you will need to write a migration.  The field in the serialized object will be populated, but when you promote a field to a column the value does not get populated by default.  So if you want to capture the count of records already managed by CR you would need to ensure the column is properly populated.

Reply:
David, thanks for bringing this up. Good catch! @ajheflin Instead of a migration we already possibly loop through policies somewhere in the datastore on start, and if we don't we could do that instead of a migration. If we go the migration route, it should be a simple one IMO, and we would have similar examples in our existing migrations.